lib/scrabble.rb

```diff
@@ -1,4 +1,8 @@
 module Scrabble
   # require your gems and classes here
-  # require_relative 'scrabble/whatevs'
+  require 'pathname'
+
+  ROOT_DIR = Pathname(__dir__).parent ## QUESTION: Is Pathname(__dir__) == Pathname.new(__dir__) ?
```

Comment:
Ruby doc was inconclusive.  Not sure if it appears to work the same way then go for it.  I feel like for the current purpose it will very quickly resolve in error if it doesn't.